ehitchhiking-ui/src/app/app.module.ts

```diff
@@ -5,20 +5,22 @@ import {AppComponent} from './app.component';
 import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
 import {FormsModule} from '@angular/forms';
 import {MainScreenModule} from './pages/main-screen/main-screen.module';
-import {MenuModule} from './shared/components/menu/menu.module';
 import {RatePassengersModalModule} from './shared/components/rate-passengers-modal/rate-passengers-modal.module';
+import {MatButtonModule} from "@angular/material";
```

Comment:
Are you sure that MatButtonModule is required here?

Reply:
remove it